cmd/piper.go

```diff
@@ -126,6 +126,7 @@ func Execute() {
 	rootCmd.AddCommand(IntegrationArtifactUpdateConfigurationCommand())
 	rootCmd.AddCommand(IntegrationArtifactGetMplStatusCommand())
 	rootCmd.AddCommand(AbapEnvironmentAssembleConfirmCommand())
+	rootCmd.AddCommand(TransportRequestCreateCTSCommand())
```

Comment:
put the transportrequest commands together

Reply:
What is the benefit? I mean we can (... as the full) team agree to having that e.g. in an alphabetical order since it looks "nicer". The more common way - as far as I can see - is having some kind of chronological order ...